mod/01_filter/set/abstract/filter.rb

```diff
@@ -141,17 +142,21 @@ def default_button_formgroup_args args
     args[:buttons] = buttons.join
   end
 
-  view :filter_form do |args|
-    formgroups = args[:formgroups] || [:name_formgroup]
+  def advance_formgroups args
```

Comment:
technically this should probably be "advanced", too.

(sorry to whine about something so minor, but I keep reading it as a verb, and that gives it a very different meaning)